src/ExpoPushNotificationsServiceProvider.php

```diff
@@ -30,6 +51,13 @@ public function boot()
      */
     public function register()
     {
-        $this->app->bind(ExpoRepository::class, ExpoFileDriver::class);
+        $driverClass = ExpoFileDriver::class;
```

Comment:
Maybe you can set the driver in the constructor to remove the duplication

Reply:
The constructor does not have access to the config value... maybe you know another way of making this cleaner?